### DISCUSSION TITLE
Not able to install on CONDA

### DISCUSSION THREAD

**Discussion by Daniil-Kuhmistrov** [NVIDIA/cuQuantum#44] [Q&A] [answered]
I'm trying to install cuQuantum on Conda. I haven't set the enviroment, but cuda is working correctly in, for instance, jupyter. However, i get this 

`conda install -c conda-forge cuquantum-python`
`Collecting package metadata (current_repodata.json): done`
`Solving environment: failed with initial frozen solve. Retrying with flexible solve.`
`Collecting package metadata (repodata.json): done`
`Solving environment: failed with initial frozen solve. Retrying with flexible solve.`

`PackagesNotFoundError: The following packages are not available from current channels:`

 ` - cuquantum-python`

`Current channels:`

  - https://conda.anaconda.org/conda-forge/win-64
  - https://conda.anaconda.org/conda-forge/noarch
  - https://repo.anaconda.com/pkgs/main/win-64
  - https://repo.anaconda.com/pkgs/main/noarch
  - https://repo.anaconda.com/pkgs/r/win-64
  - https://repo.anaconda.com/pkgs/r/noarch
  - https://repo.anaconda.com/pkgs/msys2/win-64
  - https://repo.anaconda.com/pkgs/msys2/noarch

`To search for alternate channels that may provide the conda package you're
looking for, navigate to`

    https://anaconda.org

`and use the search bar at the top of the page.`

What can possibly cause the problem and how do i solve it.

**Comment by leofang** [comment, marked as answer]
I see that your conda error message lists `win-64`, but cuQuantum does not support Windows. If you use WSL, we have user reports that indicate WSL2 would work (https://github.com/NVIDIA/cuQuantum/discussions/13), though we don't claim official support for WSL. Essentially, WSL behaves like Linux, and conda installed in a WSL environment would search for `linux-64` packages instead of `win-64` ones, so you should give it a shot.

**Reply by robliou** [reply]
Apologies if this is a noobish-type question, but are you essentially saying that to run cuQuantum, we need to install and run it from a Python distribution running on **Linux** (i.e. Ubuntu)? Thx, 

@leofang

**Comment by ronClassiq** [comment]
Is it working for arm-64?

**Reply by leofang** [reply]
For Linux + Aarch64, yes.

**Reply by ronClassiq** [reply]
TNX, and for Mac M1?

**Reply by yangcal** [reply]
We don't support Mac M1 as it does not have CUDA support
